web/themes/custom/bglobal/source/_patterns/01-molecules/hero/_hero.scss

```diff
@@ -0,0 +1,74 @@
+.hero__title.h2 {
+  font-size: 3rem;
+  line-height: 5rem;
+}
+
+.hero__subtitle {
+  line-height: 1.5;
+}
+
+.hero__container {
+  background-color: #EEEEEE;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 450px;
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  align-content: center;
+  justify-content: center;
```

Comment:
align-content and justify-content might not be necessary here, check and remove if it is not needed.